src/index.js

```diff
@@ -1,60 +1,86 @@
+/* eslint-disable import/no-unresolved */
 import './style.css';
-import { checkboxCheck, change } from './checkbox.js';
-
-const list = JSON.parse(localStorage.getItem('list')) || [
-  {
-    description: 'Say morning prayers',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Do the laundry',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Pick children from school',
-    completed: true,
-    index: 3,
-  },
-];
-
-function saveList() {
-  const myList = JSON.stringify(list);
-  localStorage.setItem('list', myList);
-}
+import add from './addTask.js';
+import rem from './removeTask.js';
+import removeSplice from './clearTask.js';
+import { setStorage } from './localStorage.js';
+import toggleTask from './toggleTask.js';
+import reorder from './reorder.js';
+
+const task = JSON.parse(localStorage.getItem('todos')) || [];
+const submitBtn = document.getElementById('push');
 
 const tasks = document.getElementById('list');
 function renderList(todos) {
   tasks.innerHTML = '';
   todos.forEach((todo) => {
     const { description, index } = todo;
     const task = document.createElement('li');
-    const desc = document.createElement('p');
+    const desc = document.createElement('span');
+    desc.setAttribute('contenteditable', true);
+    desc.classList.add('description');
+    desc.setAttribute('id', todo.index);
+    const deleteBtn = document.createElement('i');
+    deleteBtn.setAttribute('id', todo.index);
+    deleteBtn.classList.add('fas');
+    deleteBtn.classList.add('fa-trash-alt');
     const checkbox = document.createElement('input');
     checkbox.type = 'checkbox';
+    checkbox.checked = todo.completed;
     checkbox.setAttribute('data-index', index);
     desc.innerText = description;
-    task.append(checkbox, desc);
+    task.append(checkbox, desc, deleteBtn);
     tasks.append(task);
-    checkboxCheck(list, index, checkbox);
   });
 }
 
+const removeIndex = (index) => {
+  removeSplice(task, index);
+  setStorage(task);
+};
+// eslint-disable-next-line import/prefer-default-export
+export { removeIndex };
```

Comment:
[Optional]
[Conventionally](https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/exports-last.md), you should put the export declarations at the end of the file. This way it will be easy to see what is exported.